src/styles/blocks/card.css

```diff
@@ -0,0 +1,101 @@
+
+.card {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
```

Comment:
move to * selector

Reply:
When i move that to * selector, elements in card are "falling"